ClosedXML/Excel/Style/XLFillKey.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace ClosedXML.Excel
+{
+    public struct XLFillKey : IEquatable<XLFillKey>
```

Comment:
Add `[DebuggerDisplay(...)]` attributes on the new key structs. Makes it easier for debugging.

Reply:
I have just tried and evaluation of expressions turned out to be very slow making debugging extremely uncomfortable. So I implemented ```ToString()``` methods instead. It works way faster, and allows using ```XLColorKey.ToString()``` inside the ```XLFillKey.ToString()``` and so on.